index.html

```diff
@@ -331,6 +142,41 @@ <h2 class="inter">
 				</li>
 			</ul>
 		</footer>
+		<div id="modal">
+			<div id="modal-header">
```

Comment:
Kindly update your desktop popup to match the figma design better🙏

Your design:
<img width="465" alt="image" src="https://github.com/hamidazim321/Portfolio-Project/assets/37116322/92be15ef-b15f-416e-91e4-cad8a6fd55dc">

Figma design:
<img width="457" alt="image" src="https://github.com/hamidazim321/Portfolio-Project/assets/37116322/f4de2685-5781-45cb-855c-b740eece4045">

A few things to note:
- Please make sure that all elements in your modal are using the correct font size, font weight and font family as seen in the figma design